src/util/OAuth2Util.js

```diff
@@ -48,54 +48,50 @@ define(['okta', './Enums', './Errors', './Util'], function (Okta, Enums, Errors,
 
     var authClient = settings.getAuthClient(),
         options = settings.toJSON({ verbose: true }),
-        oauthParams = {},
-        extraOptions = {};
+        getTokenOptions = {};
```

Comment:
nit: is it better to call this just tokenOptions? `getTokenOptions ` sounds more like a fn name.

Reply:
Or maybe `authParams`? We use this name in other places to refer to the config passed to the authClient. The options we are building here correspond with this documentation: https://github.com/okta/okta-auth-js#authorize-options Maybe `authOptions`?

Reply:
either works